code/main.cpp

```diff
@@ -17,6 +17,7 @@ void show_usage() {
     std::cout << "Expected parameters: " << std::endl;
     std::cout << "-h help" << std::endl;;
     std::cout << "-i file with input data" << std::endl;
+    std::cout << "-c input file contains items count" << std::endl;
```

Comment:
Czyli -c jest dla inputu pełnego, a -i do takiego, w którym trzeba wygenerować profit table i liczność? Podkreśliłbym jakoś czym się te dwie opcje różnią w instrukcji.

Reply:
Albo jakoś zaznaczył, jak to jest flaga tylko

Reply:
zmieniłam opis argumentu i ogólnej instrukcji